syntheseus/interface/reaction.py

```diff
@@ -1,15 +1,13 @@
 from __future__ import annotations
 
-from abc import abstractmethod
 from dataclasses import dataclass, field
-from typing import Collection, Generic, Optional, TypeVar
+from typing import Collection, Optional, TypeVar
 
 from syntheseus.interface.bag import Bag
 from syntheseus.interface.molecule import SMILES_SEPARATOR, Molecule
 from syntheseus.interface.typed_dict import TypedDict
 
 ReactantsType = TypeVar("ReactantsType")
```

Comment:
Is this now unused?

Reply:
Good catch. Just pushed a fix to remove it.